zstash/hpss_utils.py

```diff
@@ -216,7 +216,7 @@ def add_file(
                     blocks += 1
                 # Increase the offset by the amount already saved to the tar
                 # FIXME: error: "TarFile" has no attribute "offset"
-                tar.offset += blocks * tarfile.BLOCKSIZE  # type: ignore
+                tar.offset += blocks * tarfile.BLOCKSIZE
```

Comment:
If these lines are passing in `mpy` now, we can probably remove all the error comments on the lines directly above.

Reply:
Sure, I'll take those out right now.